### PR TITLE
Update license headers & add all index.php

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: .vscode/index.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+*/
+
+header("Expires: Mon, 26 Jul 1997 05:00:00 GMT");
+header("Last-Modified: ".gmdate("D, d M Y H:i:s")." GMT");
+
+header("Cache-Control: no-store, no-cache, must-revalidate");
+header("Cache-Control: post-check=0, pre-check=0", false);
+header("Pragma: no-cache");
+
+header("Location: ../");
```
Comment (matks) [inline .vscode/index.php]: I guess it means the .vscode folder is being delivered into Addons API package too, right ? 🤔

Comment (Quetzacoalt91) [reply .vscode/index.php]: Yes, we currently don't have any control on the folders to filter when publishing on the marketplace.

**Comment by matks** [comment]
Is it also intended to replace OSL licence headers by AFL license headers ?
```
- * This source file is subject to the Open Software License (OSL 3.0)
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
```

**Comment by Quetzacoalt91** [comment]
Yes, license header was incorrect. Our modules are distributed under AFL.

**Review by matks — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
No need QA, thanks @Quetzacoalt91

**Pull request merged**